src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -72,14 +77,34 @@
     short-summary: commands to configure hostnames
 """
 
+helps['appservice web deployment'] = """
+    type: group
+    short-summary: Commands to manage web app deployments
```

Comment:
There's inconsistency here between using "Commands to manage blah..." and "Manage blah..." style. I'd recommend using one consistent style. Personally I prefer the second as it is more succinct and it's a CLI so the "Commands to..." isn't helpful. As I revise the help for the storage module, I'm moving to that style. I think most of Network is already that way as well.